_content/the-things-stack/migrate-to-v3.md

```diff
@@ -110,15 +110,11 @@ To complete the migration this way, you need to have  <a href="https://github.co
 
 ### Note about migrating an active session
 
-If your The Things Stack instance version is `3.10` or higher, you may transfer the active device session as well. This means:
+Starting from The Things Stack version `v3.12`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 
 
-- Your device will continue working on The Things Stack V3 cluster. 
-- Your OTAA device will not have to perform a new join on V3 network and you will not have to delete this device manually from V2 cluster. 
-- You will not have to delete your ABP device manually from V2 cluster.
-
-When migrating an active session, after exporting an end device from V2 cluster by using the migration tool, session and root keys will be cleared from The Things Network Stack V2 cluster by default. This means this device will no longer work in the V2 cluster. 
-
-You can disable this behavior by using the `--ttnv2.with-session=false` flag when running `ttn-lw-migrate`. In that case, you have to prevent your end device from rejoining the V2 network - for OTAA device this means changing the `AppKey` in V2 and performing a new join on V3 network, and for ABP device this means deleting it from the V2 cluster completely.
+> **Important note:** Migrating active sessions is **not available for The Things Stack Community Edition**, i.e. you will not be able to migrate active sessions if you are using The Things Network. The reason for this is that The Things Stack Community Edition uses a different `DevAddr` block than the one used by The Things Network V2. 
+>
+> However, migrating active sessions is available for other V3 deployments, including The Things Stack Open Source. Visit <a href="https://www.thethingsindustries.com/deployment/" target="_blank">Deployment Options</a> page to see how you can upgrade your plan, or <a href="mailto:support@thethingsindustries.com" target="_blank">contact our support</a>.
```

Comment:
This shouldn't be buried in notes. Assume that the audience is The Things Network community. Session migration isn't available for them. You might want to add a note for V2 SaaS customers though, but that's it.

Reply:
@johanstokking I'm confused here, isn't it that the V2 SaaS (which was a paid offering, correct me if I'm wrong) customers can migrate active sessions but only to paid offerings of V3 (and not the Community Edition)? If yes, why is that relevant for TTN community docs?

Reply:
I would suggest to create a headline about the session migration, and add this note about V2 SaaS customers. 

## Devices need to rejoin The Things Stack Community
Migrating active sessions is not possible ...